XenModel/Utils/Util.cs

```diff
@@ -366,20 +372,26 @@ public static bool TryParseIso8601DateTime(string toParse, out DateTime result)
                 DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal, out result);
         }
 
+        public static bool TryParseOtherDateTime(string toParse, out DateTime result)

```

Comment:
nit: not sure `TryParseOtherDateTime` is a good name. I'd either use `TryParseDateTime` or `TryParseNonIso8601DateTime` though I'd prefer the former

Reply:
I think TryParseNonIso8601DateTime  is better since it uses specific non iso8601 formats - TryParseDateTime should encompass all, but it doesn't. Will change it.